### PR TITLE
Add search-private-npm-packages extension

### PR DESCRIPTION
**Pull request by HsiuCL**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/search-private-npm-packages/src/search.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                shortcut={{ modifiers: ["ctrl", "shift"], key: "x" }}
+                onAction={onRemoveAllAccounts}
+              />
+            </ActionPanel>
+          }
+        />
+      ))}
+      <List.EmptyView
+        title="No Tracking Accounts Found"
+        description="Press ⌘ to add a tracking account!"
```
Comment (pernielsentikaer) [inline extensions/search-private-npm-packages/src/search.tsx:104]:
```suggestion
        description="Press ⌘ to add a tracking account!"
```
Shouldn't this be enter?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Thanks for your contribution 🔥

I have one suggestion for you 🙂

File: extensions/search-private-npm-packages/src/search.tsx (hunk 1 above)
Comment (HsiuCL) [reply extensions/search-private-npm-packages/src/search.tsx]: Thank you for your advice! I've updated the description.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**